### DISCUSSION TITLE
model punctuation

### DISCUSSION THREAD

**Discussion by Al-Fahrezi** [coqui-ai/TTS#2558] [General Q&A]
what if you want to train a model that can detect punctuation like "." and "," if there is such punctuation then the sound will be silent for a split second. is it only necessary to add those punctuation marks in the dataset?

**Comment by erogol** [comment]
In theory yes, enough to have it in the dataset.

**Reply by Al-Fahrezi** [reply]
is there any other better way

**Reply by erogol** [reply]
probably yes, but nothing off the top of my head.
